EU4toV2/Source/EU4World/Mods/Mods.h

```diff
@@ -1,30 +1,30 @@
 #ifndef EU4_MODS_H
 #define EU4_MODS_H
-
-#include "../../Configuration.h"
+#include "Configuration.h"
 #include <map>
 #include <optional>
 
 namespace EU4
 {
-	class Mods
-	{
-	public:
-		Mods(const std::vector<std::string>& usedMods, Configuration& theConfiguration);
-		
-		[[nodiscard]] std::optional<std::string> getModPath(const std::string& modName) const;
+class Mods
+{
+  public:
+	Mods() = default;
+	explicit Mods(const std::vector<std::string>& usedMods);
```

Comment:
Do note that by relying on the global nature of Configuration instead of passing it in, testing (even with test files) becomes effectively impossible.

Reply:
Configuration is PR material of its own. I tried doing that one first but it turned out very very messy. As you know every single mapper relies on it being global and uncoupling it at this stage is premature.